bin/configure-app-service.sh

```diff
@@ -41,14 +41,14 @@ echo "  APP_NAME=$APP_NAME"
 echo "  ENVIRONMENT=$ENVIRONMENT"
 echo
 
-cp $MODULE_DIR/example.tfvars $TF_VARS_FILE
-sed -i.bak "s/<ENVIRONMENT>/$ENVIRONMENT/g" $TF_VARS_FILE
-sed -i.bak "s/<TF_STATE_BUCKET_NAME>/$TF_STATE_BUCKET_NAME/g" $TF_VARS_FILE
-sed -i.bak "s|<TF_STATE_KEY>|$TF_STATE_KEY|g" $TF_VARS_FILE
-sed -i.bak "s/<REGION>/$REGION/g" $TF_VARS_FILE
-rm $TF_VARS_FILE.bak
+cp "$MODULE_DIR/example.tfvars" "$TF_VARS_FILE"
+sed -i.bak "s/<ENVIRONMENT>/$ENVIRONMENT/g" "$TF_VARS_FILE"
+sed -i.bak "s/<TF_STATE_BUCKET_NAME>/$TF_STATE_BUCKET_NAME/g" "$TF_VARS_FILE"
+sed -i.bak "s|<TF_STATE_KEY>|$TF_STATE_KEY|g" "$TF_VARS_FILE"
+sed -i.bak "s/<REGION>/$REGION/g" "$TF_VARS_FILE"
```

Comment:
May want to consider using `envsubst` for replacing env variables in template files - https://www.gnu.org/software/gettext/manual/html_node/envsubst-Invocation.html

Reply:
Ok, I'll create a separate ticket for that, since it would complicate this PR if we mixed functional changes with linter changes https://github.com/navapbc/template-infra/issues/421